analyzer/src/main/kotlin/managers/Npm.kt

```diff
@@ -462,8 +462,8 @@ open class Npm(
         val json = jsonMapper.readTree(packageJsonFile)
 
```

Comment:
Should we also add better error handling?

```suggestion
        if (!json.has("name") || !json.has("version")) {
            log.warn("name and version are required fields in the package.json")
        }
```

Reply:
I decided to create separate checks for name and version.